resources/assets/js/app/services/AuthService.js

```diff
@@ -0,0 +1,76 @@
+import jwtDecode from 'jwt-decode';
+import _ from 'lodash';
+
+const storage = sessionStorage,
+    tokenKeyName = 'jwt',
+    redirect = {
+        authPath: 'login',
+        rootPath: 'dashboard',
+    };
+
+export const getAuthToken = () => {
+    return storage[tokenKeyName];
+};
+
+export const isAuthorized = () => {
+    return !!getAuthToken();
+};
+
+export const initSession = (token) => {
+    storage.setItem(tokenKeyName, token);
+};
+
+export const destroySession = () => {
+    storage.removeItem(tokenKeyName);
```

Comment:
remove token from global state in addition to sessionStorage. you need anything that will let app know that auth state was changed so you could dynamically reflect auth status in the app